crates/noirc_evaluator/src/ssa_refactor/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -554,6 +554,41 @@ impl AcirContext {
         Ok(witnesses)
     }
 
+    /// Returns a vector of `AcirVar`s constrained to be the decomposition of the given input
+    /// over given radix.
+    ///
+    /// The `AcirVar`s for the `radix_var` and `limb_size_var` must be a constant
+    ///
+    /// TODO: support radix larger than field modulus
+    pub(crate) fn radix_decompose(
```

Comment:
I think we only use powers of 2 currently in our codebase, we can just support that and then when we implement brillig, it will be done in Brillig code instead